### ISSUE TITLE
Can this be used to bind to TWebBrowser javascript?

### ISSUE THREAD

**Issue by GaNacereddine** [grijjy/DelphiDuktape#3]
it is possible to register delphi functions and call them from JavaScript in a TWebBrowser app?

Thanks

**Comment by skamradt** [comment]
No, the context is wrong.  Your TWebBrowser javascript is being run by the browser, not the Duktape engine.  You would have to register your delphi functions with the javascript engine inside the TWebBrowser application.
